source/renderer/app/api/api.js

```diff
@@ -456,6 +457,9 @@ export default class AdaApi {
           throw new NotEnoughFundsForTransactionError();
         }
       }
+      if (error.message === 'TooBigTransaction') {
```

Comment:
@daniloprates please add this error handling to `createTransaction` too 🙏

Reply:
Ok

Reply:
✅